versions.props

```diff
@@ -91,4 +93,19 @@
     <OracleEFCoreVersion>5.21.1</OracleEFCoreVersion>
     <PomeloEFCoreVersion>5.0.0-alpha.2</PomeloEFCoreVersion>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(TargetFramework)' == 'net6.0'">
+    <AspNetCoreVersion>6.0.0-rc.2.*</AspNetCoreVersion>
```

Comment:
Do these need to be rcs?

Reply:
Probably not anymore, we were being held back by EF Core packages for MySQL, Postgres and Oracle